themes/default/items/characters.json5

```diff
@@ -0,0 +1,13 @@
+[
+    {
+        id: "bobsonDugnut",    //must be unique - kebab case it? If we stick to alphanumerics it can be pulled out directly.
+        title: "Bobson Dugnut",
+        description: "The greatest baseball player to ever live."
+    },
+    {
+        id: "sleveMcDichael",    //must be unique - kebab case it? If we stick to alphanumerics it can be pulled out directly.
+        title: "Sleve McDichael",
+        description: "He could have been the greatest if it wasn't for his bad luck with injuries."
+    },
+
+]
```

Comment:
can we add @AnthonyGoodchild  to the character list

Reply:
Liron Man 🦁